cmake/BLTMacros.cmake

```diff
@@ -219,8 +221,10 @@ macro(blt_add_target_link_flags)
         endif()
         # append new flag
         set(_LINK_FLAGS "${arg_FLAGS} ${_LINK_FLAGS}")
+        # LINK_FLAGS property _must_ be a string
```

Comment:
Are there ever cases where link flags could legitimately contain a semicolon?

Reply:
I don't know, but that does concern me.

Reply:
Wouldn't the command line parser think it was two commands then... IE "rm -rf *; cmake .."

Reply:
I think you mean `rm -rf *cmake*` 😄

Reply:
You could pass a quoted argument with a semicolon though? Like `"blah;blah;blah"` - this regex would trash that.

Reply:
@davidbeckingsale -- I was going to say the same thing 
(e.g. https://stackoverflow.com/a/20233998), 
but that does help with my concern.

Is it worth mentioning the semicolon stripping in the documentation, so that if it does come up, users can have some idea about what caused it ?

Reply:
I don't think I have ever seen a case where semicolon's would be used.  
b/c semicolon would terminate the current command and truncate it as @white238  states.

Reply:
Okay, I updated the documentation like @kennyweiss suggested.